duckduckgo/iOS
# make it one step to add a bookmark

Task/Issue URL: https://app.asana.com/0/392891325557410/1114659762004292
Tech Design URL:
CC:

**Description**:

Remove prompts for adding bookmarks and favourites.  Also improve handling of page title so that it shows correctly in newly added bookmarks and on tabs.

**Steps to test this PR**:

1.  Add a bookmark and favourite from the menu 
1. Expected: no prompt, toast shows bookmark/favorite added

1. Navigate to a website with a title (e.g. cnn.com) 
1. Navigate to a page without a title (e.g. https://falkirkrpg.org.uk/noxwall/frame.html)
1. Expected: Tabs show domain for the tab title
1. Add bookmark
1. Expected: Bookmark uses domain name for the title
1. Use back button to navigate back to CNN
1. Expected: Tabs show correct title for CNN 
1. Add bookmark
1. Expected: Bookmark uses page title for the title

1. Search for someone on twitter
1. Expected: Tabs shows DuckDuckGo search as page title
1. Navigate to Twitter link
1. Expected: Tabs shows page title for the Twitter link
1. Navigate to Twitter hashtag
1. Expected: Tabs correctly shows page title 

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
